schema/plugin.json

```
@@ -41,6 +41,12 @@
       "description": "Number of (most recent) commits shown in the history log",
       "default": 25
     },
+    "refreshIfHidden": {
+      "type": "boolean",
+      "title": "Refresh if Git tab is hidden",
+      "description": "Whether to refresh even if the Git tab is hidden",
```

`refresh` doesn't have a specific meaning in terms of Git, so it may be less clear what this means someone who hsan't developed this extension. This maybe clarifies it bit?

```suggestion
      "title": "Refresh if the Git tab is hidden",
      "description": "Whether to check Git status when the Git tab is not visible. Choose `false` for higher performance.",
```

---

Thanks for the suggestion. It sounds clearer.